.cicd/pipeline.yml

```diff
@@ -147,6 +147,150 @@ steps:
 
   - wait
 
+  - label: ":aws: Amazon_Linux 2 - EOSIO.CONTRACTS Build"
+    command:
+      - "buildkite-agent artifact download build.tar.gz . --step ':aws: Amazon_Linux 2 - Build' --agent-access-token $$BUILDKITE_AGENT_ACCESS_TOKEN && tar -xzf build.tar.gz"
+      - "./.cicd/eosio-contracts-build.sh"
+    env:
+      IMAGE_TAG: "amazonlinux-2"
+    agents:
+      queue: "automation-eks-eos-tester-fleet"
+    timeout: ${TIMEOUT:-10}
+    skip: ${SKIP_AMAZON_LINUX_2}${SKIP_LINUX}${SKIP_UNIT_TESTS}
```

Comment:
Even `*** - EOSIO.CONTRACTS Build` are in the same step as unit tests, I think it's better to replace `SKIP_UNIT_TESTS` with a variable like `SKIP_WASM_METRICS`, so that it's possible to skip WASM metrics tests and unit tests separately.

Reply:
@bogniq, yes I added SKIP_WASM_METRICS as key to skip all the wasm size related steps in the pipeline.